packages/fluent-theme/src/components/Theme.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { type ReactNode } from 'react';
+import { useStyles } from '../styles';
+
+const styles = {
+  webchat__theme: {
+    '--colorNeutralForeground1': '#242424',
```

Comment:
Probably okay for now.

But we need to know how to make this... web devs are very unlikely to tune each value.

Unlike Fluent, which has a set of palette based on a limited number of accent colors. Our customer has their own accent color.

Reply:
Also, think about if the customer can (kind of) easily copy Fluent palette over. Or maybe we should offer all Fluent palettes.

Reply:
I'm going to allow falling back to the FluentUI palette if it is available at the root. Looks like all we need is the following change:
```
--webchat-colorNeutralForeground1: var(--colorNeutralForeground1, #242424);
```

Reply:
@compulim need you word here to make the change

Reply:
Discussed offline. Implemented